src/libs_3rdparty/QSpec/src/utils/GTUtilsDialog.h

```diff
@@ -25,6 +25,7 @@
 #include <core/CustomScenario.h>
 
 #include <QDialogButtonBox>
+#include <QProcess>
```

Comment:
Looks like we do not need this include here.

Reply:
fixed